eras/conway/impl/src/Cardano/Ledger/Conway/Governance/Procedures.hs

```diff
@@ -327,7 +327,7 @@ instance EraPParams era => EncCBOR (GovActionState era) where
         !> To gasExpiresAfter
 
 instance OMap.HasOKey GovActionId (GovActionState era) where
-  okeyL = lens gasId $ \gas gi -> gas {gasId = gi}
```

Comment:
In conway cabal file, `cardano-data >=1.2.3,` should  get bumped to `1.4` if you bump the latter, i think